pkg/config/config.go

```diff
@@ -32,6 +32,7 @@ import (
 
 	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
 	acktags "github.com/aws-controllers-k8s/runtime/pkg/tags"
+	ackrtutil "github.com/aws-controllers-k8s/runtime/pkg/util"
```

Comment:
you can just name this alias `ackutil` since we're already in the runtime package.